spec/_pages/who_uses_login.md_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+RSpec.describe 'who_uses_login.md' do
+  let(:doc) { Nokogiri::HTML(file_at('/who-uses-login')) }
```

Comment:
A lot of pages in this site go un-tested. There are a few pages that have their own unique requirements, but for the most part I think we should be running the same specs against every generated HTML file.... seems like a good task for another PR